src/components/application_manager/src/message_helper/message_helper.cc

```diff
@@ -1344,6 +1344,29 @@ smart_objects::SmartObjectList MessageHelper::CreateAddCommandRequestToHMI(
   return requests;
 }
 
+void MessageHelper::CreateUnsubscribeVehicleDataMessageForHMI(
+    smart_objects::SmartObjectSPtr& message_to_hmi,
+    const VehicleInfoSubscriptions& vehicle_data,
+    const application_manager::ApplicationSharedPtr& app) {
+  message_to_hmi =
+      CreateMessageForHMI(hmi_apis::messageType::request, app->app_id());
+  (*message_to_hmi)[strings::params][strings::function_id] =
+      hmi_apis::FunctionID::VehicleInfo_UnsubscribeVehicleData;
+  smart_objects::SmartObject msg_params =
```

Comment:
@IGapchuk good place for auto

Reply:
Done in 73c67bf4f764f384b64cc45f9a7a3087eff31f1f.